addons/l10n_it_ddt/models/account_invoice.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    def _get_ddt_values(self):
+        ddt_dict = {}
+        line_count = 1
+        for line in self.invoice_line_ids.filtered(lambda l: not l.display_type):
+            for picking in line.picking_ids:
+                if picking not in ddt_dict:
+                    ddt_dict[picking] = []
+                ddt_dict[picking].append(line_count)
+            line_count += 1
+        for picking in ddt_dict:
+            if len(ddt_dict[picking]) == line_count-1:
+                ddt_dict[picking] = []
```

Comment:
Because it has everything on it?  Maybe put a comment to explain.

Reply:
it done.